Core/include/Acts/Surfaces/SurfacesError.hpp

```diff
@@ -0,0 +1,55 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <iostream>
+#include <string>        // for string printing
+#include <system_error>  // bring in std::error_code et al
+
+namespace Acts {
+// This is the custom error code enum
+enum class SurfacesError {
+  LocalToGlobalFailed = 1
+};
```

Comment:
The naming sounds a bit strange. Maybe just `SurfaceError`.

I would also suggest to make the error flag more explicit, e.g. `GlobalPositionNotOnSurface`.

Reply:
Ok, will change this.

Reply:
Done.